applications/vessel-history/src/classes/vessel.class.ts

```diff
@@ -40,31 +47,41 @@ export class VesselInfo {
     }
     const dataValues =
       tmtValue?.map((historicValue: AnyValueList) => {
-        console.log(historicValue.firstSeen)
         const value: HistoricValue = {
           data: historicValue.value,
-          start: historicValue.firstSeen ? new Date(historicValue.firstSeen) : null,
-          end: historicValue.endDate ? new Date(historicValue.endDate) : null,
+          start: historicValue.firstSeen
+            ? DateTime.fromISO(historicValue.firstSeen, { zone: 'UTC' })
```

Comment:
Seems we have this function repeated in several places, what about creating a helper like `isoToUtcDate` to reduce repetition?

Reply:
Most of the `vessel.class.ts` will be changed in another branch `vessel-history/VH-82-refactor-thunks`